cmd/k8s-operator/ingress-for-pg.go

```diff
@@ -783,55 +783,83 @@ func (a *HAIngressReconciler) numberPodsAdvertising(ctx context.Context, pgName
 	return count, nil
 }
 
+const ownerAnnotation = "tailscale.com/owner"
```

Comment:
Suggestion:

```suggestion
const ownerAnnotation = "tailscale.com/owner-references"
```
(makes it clearer what it is + consistent with kube)